Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM           golang:latest
+MAINTAINER     no-reply@noms.io
```

Comment:
noms.io is not available right now. Can we send this to a github email alias or something?

Reply:
Oh, the field is deprecated anyway. We can just remove it.